### PR TITLE
Added Hasura

### PR DESCRIPTION
**Pull request by akashkj**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](../CONTRIBUTING.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +121,22 @@ None right now.
 - **Issues**: [#hacktoberfest](https://github.com/search?&q=user%3Aappwrite+label%3Ahacktoberfest+type:issue&state=open&type=Issues) or any issue you like to help with.
 - **Notes**: Follow [Appwrite blog](https://medium.com/appwrite-io) and [Discord community](https://appwrite.io/discord) for more information and help.
 
+#### **Hasura** (Stickers, T-shirt)
+
+- **Swag**: T-shirt, stickers
+- **Requirements**:
+  - Submit a PR with the label hacktoberfest: Swags
```
Comment (hugovk) [inline README.md:128]:
```suggestion
  - Submit a PR with the label hacktoberfest: Swag
```

File: README.md [hunk 2]
```diff
@@ -56,6 +56,24 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Please, fill out this [DX Heroes Hacktoberfest2020 Participant Form](https://forms.gle/o3sBqsjr4sYLQgPh6) first. Then just visit one of our open source projects - [DX Knowledge Base](https://github.com/DXHeroes/knowledge-base-content) or [DX Scanner](https://github.com/DXHeroes/dx-scanner), make a pull request and compete with other contributors in the friendly competition to earn your place in the Leaderboard of Contributors.
 - **Notes**: If you have any questions or need help with anything, join our [Community Slack](https://bit.ly/slack_developer_experience). We will gladly help you. Happy Hacktoberfest 2020!
 
+### H
+
+#### **Hasura**
+
+- **Swag**: T-shirt, stickers
+- **Requirements**:
+  - Submit a PR with the label hacktoberfest: Swags
```
Comment (hugovk) [inline README.md:65]:
```suggestion
  - Submit a PR with the label hacktoberfest: Swag
```

**Review by hugovk — CHANGES_REQUESTED** [review]
"Swag" is a mass noun. And can you be more specific than "Swag"?

File: README.md (hunk 1 above)
Comment (akashkj) [reply README.md:128]: Page doesn't mention anything specifc as of now. They have mentioned though that they will announce in future. Any suggestion?

**Comment by crweiner** [comment]
Hey @akashkj and @hugovk! Thanks for working together to get Hasura on the 2020 Hacktoberfest Swag List! I greatly appreciate your time and teamwork! 👍 

Has Hasura made it clear if they are just offering the most-requested swag item from their Twitter poll or if they're offering all 3?

**Comment by akashkj** [comment]
@crweiner I don't think they have clarified that yet.

**Comment by crweiner** [comment]
Hi @akashkj and @hugovk! You both rock! Thank you for adding Hasura to the 2020 Hacktoberest Swag List and making sure that this PR is formatted properly. 🎉 🎉  Have a marvelous Hacktoberfest!!

you rock

**Pull request merged**